src/wallet/rpcdump.cpp

```diff
@@ -33,29 +33,33 @@ UniValue importwallet_impl(const UniValue& params, bool fHelp, bool fImportZKeys
 
 enum SpendingKeyAddResult {
     KeyAlreadyExists,
-    Added,
-    NotAdded,
+    KeyAdded,
+    KeyNotAdded,
```

Comment:
If you rebase this PR to move `AddSpendingKeyToWallet` into `wallet.h`, please refactor this into the previous commit (that added `SpendingKeyAddResult`).